### DISCUSSION TITLE
Achievements View - Too big

### DISCUSSION THREAD

**Discussion by sunho** [community/community#18148] [Other Feature Feedback, Questions, & Ideas]
It's a nice addition but I don't like it taking too much space on profile.

**Comment by cipherwithadot** [comment]
i agree, it could be just a little bit smaller

**Comment by doc-code-hub** [comment]
Agree, really too big...

Considering also that the chosen graphics are not to my liking, it weighs me even more...

A more understated size would be greatly appreciated.

**Comment by rawleyfowler** [comment]
Yeah it would be nice if they were smaller on your page, then have some sort of other view for them where they're expanded.

**Reply by odnmalau** [reply]
It would be nice!

**Comment by bailey** [comment]
Thank you all for this feedback! We're iterating on how to best surface them on the profile, so we'll definitely take this into consideration.
